index.js

```diff
@@ -135,7 +141,7 @@ File.prototype.inspect = function() {
   var inspect = [];
 
   // Use relative path if possible
-  var filePath = (this.base && this.path) ? this.relative : this.path;
+  var filePath = this.path ? this.relative : this.path;
```

Comment:
why `this.path` at the end of the ternary?  Wouldn't it be something falsey always?  Could probably get away with hardcoding an empty string or using undefined/null instead.  1 less getter call.